action.yml

```diff
@@ -109,6 +116,13 @@ runs:
     - name: Docker Build
       shell: bash
       run: ${{ github.action_path}}/scripts/docker_build
+    
+    - name: Run Test
+      if: ${{env.PLATFORM == 'linux/amd64'}}
```

Comment:
Given making the `smoke_test` input optional as mentioned above, we should also gate running this step on whether any smoke test was actually defined, otherwise it will show up even when no tests are run, which experience tells me will eventually confuse someone and take up some of their time.

Reply:
I added a check where it will only run if smoke test is defined.